exasol_integration_test_docker_environment/lib/base/luigi_log_config.py

```diff
@@ -52,16 +52,16 @@ def restore_logger(logger_creator: Callable[[], logging.Logger]):
     }
     yield
     after_logger = logger_creator()
-    after_logger.level = logger_info[LOG_LEVEL]
-    after_logger.handlers = logger_info[HANDLERS]
-    after_logger.filters = logger_info[FILTERS]
-    after_logger.propagate = logger_info[PROPAGATE]
+    after_logger.level = logger_info[LOG_LEVEL] # type: ignore
```

Comment:
Maybe we should use data class instead of a dictionary here, that could be properly typed

Reply:
Done